websites/K/KQM/presence.ts

```diff
@@ -0,0 +1,134 @@
+const presence = new Presence({
+	clientId: "1215004822676312174",
+});
+
+function cleanAndCapitalize(string: string): string {
+	return string
+		.replace(/[_-]/g, " ")
+		.split(" ")
+		.map(word => word.charAt(0).toUpperCase() + word.substring(1))
+		.join(" ");
+}
+
+const browsingTimestamp = Math.floor(Date.now() / 1000),
+	characterList = [
+		"amber",
+		"barbara",
+		"beidou",
+		"bennett",
+		"chongyun",
+		"diluc",
+		"diona",
+		"eula",
+		"fischl",
+		"ganyu",
+		"hu_tao",
+		"jean",
+		"kaeya",
+		"kaedehara_kazuha",
+		"keqing",
+		"klee",
+		"lisa",
+		"mona",
+		"ningguang",
+		"noelle",
+		"qiqi",
+		"razor",
+		"rosaria",
+		"sucrose",
+		"tartaglia",
+		"anemo-traveler",
+		"geo-traveler",
+		"venti",
+		"xiangling",
+		"xiao",
+		"xingqiu",
+		"xinyan",
+		"yanfei",
+		"zhongli",
+		"albedo",
+		"kujou_sara",
+		"raiden_shogun",
+		"kamisato_ayaka",
+		"yoimiya",
+		"electro-traveler",
+		"sangonomiya_kokomi",
+		"sayu",
+		"aloy",
+		"thoma",
+		"itto",
+		"gorou",
+		"yunjin",
+		"shenhe",
+		"yae",
+		"ayato",
+		"shinobu",
+		"dendro-traveler",
+		"heizou",
+		"tighnari",
+		"collei",
+		"dori",
+		"candace",
+		"cyno",
+		"nilou",
+		"nahida",
+		"layla",
+		"faruzan",
+		"wanderer",
+		"alhaitham",
+		"yaoyao",
+		"dehya",
+		"mika",
+		"baizhu",
+		"kaveh",
+		"kirara",
+		"hydro-traveler",
+		"lyney",
+		"lynette",
+		"freminet",
+		"neuvillette",
+		"wriothesley",
+		"furina",
+		"charlotte",
+		"navia",
+		"chevreuse",
+		"xianyun",
+		"gaming",
+	];
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/ReqQMXo.png",
```

Comment:
```suggestion
			largeImageKey: "https://i.imgur.com/zPaoj5v.png",
```
its 404 again xD

Reply:
imgur just sucks

Reply:
Yeah, it's been affecting a lot of images recently.